src/docker/workflows/cp-schema-registry.ts

```diff
@@ -165,16 +162,11 @@ export class ConfluentPlatformSchemaRegistryWorkflow extends LocalResourceWorkfl
     this.progress = progress;
     this.imageTag = getLocalSchemaRegistryImageTag();
 
-    const repoTag = `${ConfluentPlatformSchemaRegistryWorkflow.imageRepo}:${this.imageTag}`;
-    const containerListRequest: ContainerListRequest = {
-      all: true,
-      filters: JSON.stringify({
-        ancestor: [repoTag],
-        label: [MANAGED_CONTAINER_LABEL],
-      }),
-    };
-    const existingContainers: ContainerSummary[] =
-      await getContainersForImage(containerListRequest);
+    const existingContainers: ContainerSummary[] = await getLocalResourceContainers(
+      this.imageRepo,
+      this.imageTag,
+      { onlyExtensionManaged: true, statuses: [ContainerStateStatusEnum.Running] },
+    );
```

Comment:
This addresses a possible-but-unreported bug where we could offer the "Stop" action for a period of time before the event listener indicates an SR container is actually stopped. So now we're explicitly filtering for the `running` status similar to the existing behavior that checks Kafka containers before stopping: https://github.com/confluentinc/vscode/blob/473a85421c03ecd3cafc64f2243c5fbc1c3f2f25/src/docker/workflows/confluent-local.ts#L169-L175